main.go

```diff
@@ -145,7 +145,7 @@ func serviceAddress() string {
 		return svcAddr
 	}
 
-	return "mesherylocal.layer5.io"
+	return "localhost"
```

Comment:
I checked some of adapters and all of them are using `mesherylocal.layer5.io` as service address. Why we need to change this on `meshery-istio`?

Reply:
Great question, @Aisuko. I was just starting here on this adapter somewhat randomly as the first adapter to receive this change. The need for use of `mesherylocal.layer5.io` as an alternate to `localhost` has dissipated. Ultimately, we should be able to remove this note as well - https://docs.meshery.io/project/contributing/contributing-ui#running-meshery-from-ide

Reply:
@leecalcote Looks good. Is there any test case can make sure this work well?  If sure. I can help do PR for other adapters and Helm charts.

Reply:
@navendu-pottekkat @Revolyssup will you confirm that the e2e test cases (that ran here - https://github.com/meshery/meshery-istio/runs/4659835134?check_suite_focus=true) covers this change?

Reply:
@Aisuko, yes, please. Your help in propagating this change to the rest of the adapters would be excellent.

Reply:
@leecalcote No problem.

Reply:
@leecalcote  in these tests, adapter is build from dockerfile, and it has service address as 
https://github.com/meshery/meshery-istio/blob/d89889fb6dc8f50f817399f363328fed506ece40/Dockerfile#L30
SO no probs

Reply:
Ok, good deal, @Aisuko. 👍 Approved for merge?